build.gradle

```diff
@@ -90,6 +90,7 @@ dependencies {
     implementation "net.masterthought:cucumber-reporting:$masterThoughtVersion"
     implementation "org.aspectj:aspectjrt:$project.aspectJVersion"
     implementation "in.specmatic:specmatic-core:${project.specmaticVersion}"
+    implementation 'io.github.bonigarcia:webdrivermanager:5.5.3'
```

Comment:
specify the version like other dependencies

Reply:
Updated